e2e/fixtures/ssr-basic/waku.config.ts

```diff
@@ -3,11 +3,10 @@
 import { defineConfig } from 'waku/config';
 
 export default defineConfig({
-  unstable_viteConfigs: {
-    'dev-main': () => ({
-      optimizeDeps: {
-        exclude: ['@ai-sdk/rsc'],
-      },
-    }),
+  vite: {
+    // FIXME How can we limit this config for only mode=development?
```

Comment:
@hi-ogawa 

> vite.config.ts is completely ignored.

That's totally fine and preferred if it's intentional.
But, how do we deal with this case? "for only mode=development"

Reply:
moving the discussion to https://github.com/wakujs/waku/pull/1634/files#r2283756210